src/main/developer-tools.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Installs Electron developer tools in the development build.
+ * The dependency is not bundled to the production build.
+ */
+export const installDeveloperTools = async () => {
+  if (process.env.NODE_ENV === 'development') {
```

Comment:
`isDevelopment` variable is available in `common/vars.ts`

Reply:
If we use the environment variable through a variable assignment, then webpack can't optimize the build. In other words if `isDevelopment` is used, the code inside the if branch is included in the production build. However it _appears_ the dependency itself (`electron-devtools-installer`) is excluded in both cases, but I'd still keep using process.env directly to make sure future changes will exclude the code too.